contracts/schemes/GenericSchemeMultiCall.sol

```diff
@@ -152,6 +152,8 @@ contract GenericSchemeMultiCall is VotingMachineCallbacks, ProposalExecuteInterf
             } else {
                 (success, genericCallReturnValue) =
                 controller.genericCall(proposal.contractsToCall[i], callData, avatar, proposal.values[i]);
+                /* Whole transaction will be reverted if at least one call fails*/
+                require(success, "Proposal call failed");
```

Comment:
please move that out of the` }` so it will include the `externalTokenApproval` as well .

Reply:
@orenyodfat just updated.